tests/app/platform_admin/test_rest.py

```diff
@@ -0,0 +1,115 @@
+import uuid
+
+import pytest
+
+from app.models import (
+    ApiKey,
+    Complaint,
+    EmailBranding,
+    InboundNumber,
+    InboundSms,
+    Job,
+    LetterBranding,
+    Notification,
+    Organisation,
+    ProviderDetails,
+    Service,
+    ServiceCallbackApi,
+    ServiceContactList,
+    ServiceDataRetention,
+    ServiceEmailReplyTo,
+    ServiceInboundApi,
+    ServiceSmsSender,
+    Template,
+    TemplateFolder,
+    User,
+)
+from app.platform_admin.rest import (
+    FIND_BY_UUID_EXTRA_CONTEXT,
+    FIND_BY_UUID_MODELS,
+)
+from tests.app.db import (
+    create_complaint,
+    create_email_branding,
+    create_inbound_number,
+    create_inbound_sms,
+    create_letter_branding,
+    create_service_callback_api,
+    create_service_contact_list,
+    create_service_data_retention,
+    create_service_inbound_api,
+    create_service_with_inbound_number,
+    create_template_folder,
+)
+
+
+class TestFindByUUID:
+    def load_sample_data(self, request, model):
+        if model == Organisation:
+            request.getfixturevalue("sample_organisation")
+        elif model == Service:
+            request.getfixturevalue("sample_service")
+        elif model == Template:
+            request.getfixturevalue("sample_template")
+        elif model == Notification:
+            request.getfixturevalue("sample_notification")
+        elif model == EmailBranding:
+            create_email_branding()
+        elif model == LetterBranding:
+            create_letter_branding()
+        elif model == User:
+            request.getfixturevalue("sample_user")
+        elif model == ProviderDetails:
+            request.getfixturevalue("sms_providers")
+        elif model == ServiceEmailReplyTo:
+            request.getfixturevalue("notify_service")
```

Comment:
why's this returning the notify service? 🤔  i think i'd rather we created a sepcific reply to object on sample_service as we've done with some of the models below

Reply:
TBH I just picked the first fixture I saw with a reply-to-email 🙄  Let's change it yes